dev/ts_image.py

```diff
@@ -36,89 +45,48 @@
                     help='Input excess model FITS file name')
 parser.add_argument('--threshold', type=float, default=None,
                     help="Minimal required initial (before fitting) TS value, that the fit is done at all.")
-parser.add_argument('--overwrite', action='store_true',
-                    help='Overwrite existing output file?')
+parser.add_argument('--clobber', action='store_true',
```

Comment:
I've opted for consistent use of `overwrite` in `Gammapy` instead of `clobber`.
I was 50:50 on this, but `astropy.table.Table` uses `overwrite` and I think the plan is to use it consistently in Astropy, even if now `astropy.io.fits` uses `clobber`.
OK?